.werft/build.ts

```diff
@@ -608,6 +608,8 @@ export async function deployToDevWithInstaller(deploymentConfig: DeploymentConfi
         werft.fail(installerSlices.INSTALLER_POST_PROCESSING, err);
     }
 
+    installMonitoring(deploymentConfig.namespace, 9100, deploymentConfig.domain, true);
```

Comment:
I think needs to be guarded by withVM right? Otherwise we'd try to install monitoring in core-dev when you use the installer which means we'd almost always install monitoring as the installer is the default.

We could also move this some line [316 - 351](https://github.com/gitpod-io/gitpod/blob/main/.werft/build.ts#L316-L351) so that installing monitoring is part of "creating the VM" as disconnected from the Gitpod installation ☺️